### DISCUSSION TITLE
What does Makefile output mean?

### DISCUSSION THREAD

**Discussion by tisonkun** [tensorchord/envd#1498] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/18818196/220532039-aeeeb3a4-bf93-46bf-80fb-aa1189ce0300.png)

It looks incomprehensible.

cc @kemingy @gaocegege

**Comment by tisonkun** [comment, marked as answer]
It seems the output of `go build -v` which means "print the names of packages as they are compiled", while I may prefer just be silent since developers can gain little from such output. WDYT?

**Reply by gaocegege** [reply]
SGTM, we could remove -v here.
